index-advisor.md

```diff
@@ -0,0 +1,222 @@
+| max_num_index     | 5     | The maximum number of indexes to recommend.             |
+| max_index_columns | 3     | The maximum number of columns in an index.              |
+| max_num_query     | 1000  | The maximum number of queries to recommend indexes.     |
+| timeout           | 30s   | The timeout of index advisor.                           |
++-------------------+-------+---------------------------------------------------------+
+4 rows in set (0.00 sec)
+```
+
+To modify an option, use the `RECOMMEND INDEX SET` command. For example, to change the `timeout` option:
+
+```sql
+mysql> RECOMMEND INDEX SET timeout='20s';
+Query OK, 1 row affected (0.00 sec)
+```
+
+### Limitations
+
+The index recommendation feature has the following limitations, which will be addressed in future releases:
+
+- It does not support prepared statements. The `RECOMMEND INDEX RUN` command cannot recommend indexes for queries executed through the `Prepare` and `Execute` protocol.
+- It does not provide recommendations for deleting indexes. Future updates will include logic for [removing indexes](#remove-unused-indexes) in the `RECOMMEND` command.
+- A user interface (UI) for the Index Advisor is not yet available but will be introduced in the future.
+
+## Remove unused indexes
+
+For v8.0.0 or later versions, you can identify inactive indexes in your workload using `sys.schema_unused_indexes` and `information_schema.tidb_index_usage`. Removing these indexes can save storage space and reduce overhead. For production systems, it is highly recommended to make the target indexes invisible first and observe the impact for one complete business cycle before permanently removing them.
+
+### Use `sys.schema_unused_indexes`
+
+The [`sys.schema_unused_indexes`](/sys-schema/sys-schema-unused-indexes.md) view identifies indexes that have not been used since the last startup of all TiDB instances. This view, based on system tables containing schema, table, and column information, provides the full specification for each index, including schema, table, and index names. You can query this view to decide which indexes to make invisible or delete.
+
+> **Warning:**
+>
+> Because the `sys.schema_unused_indexes` view shows unused indexes since the last startup of all TiDB instances, ensure that the TiDB instances have been running long enough. Otherwise, the view might show false candidates if certain workloads have not yet run. Use the following SQL query to identify the uptime of all TiDB instances.
+>
+> ```sql
+> SELECT START_TIME,UPTIME FROM INFORMATION_SCHEMA.CLUSTER_INFO WHERE TYPE='tidb';
+> ```
+
+### Use `information_schema.tidb_index_usage`
+
+The [`information_schema.tidb_index_usage`](/information-schema/information-schema-tidb-index-usage.md) table provides metrics such as selectivity buckets, last access time, and rows accessed. The following examples show queries to identify unused or inefficient indexes based on this table:
+
```

Comment:
The `CLUSTER_TIDB_INDEX_USAGE` table is used in the following examples, while  the`TIDB_INDEX_USAGE` table is mentioned before the examples. Can we use one of them consistently?

Reply:
Updated, PTAL